src/main/java/Zap.java

```diff
@@ -0,0 +1,236 @@
+
+    /**
+     * `addTask` method adds new task to the list of tasks
+     * creates a new `Task` object with description,
+     * adds it to the `tasks` list
+     */
+    private static void addTask(String taskDescription) {
+        if (taskDescription.isEmpty()) {
+            System.out.println("Woi. You think I robot then can waste my time >:(");
+        } else {
+            tasks.add(new Task(taskDescription));
+            System.out.println("____________________________________________________________");
+            System.out.println(" added: " + taskDescription);
+            System.out.println("____________________________________________________________");
+        }
+    }
+
+    private static void addTodoTask(String userCommand) {
+
+        String taskDescription = userCommand.substring(4).trim();
+        tasks.add(new TodoTask(taskDescription));
+        System.out.println("____________________________________________________________");
+        System.out.println(" Got it. Zappy boy added this task:");
+        System.out.println("   [T][ ]  " + taskDescription);
+        if (tasks.size() == 1) {
+            System.out.println(" Now you have 1 task in the list.");
+        } else {
+            System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void addDeadline(String userCommand) {
+        String[] descParts = userCommand.split("deadline");
+        String[] deadlineParts = descParts[1].split("/by", 2);
+
+        if (deadlineParts.length != 2) {
+            System.out.println("Invalid deadline format. Please use '/by' to specify day.");
+            return;
+        }
+
+        String description = deadlineParts[0];
+        String deadline = deadlineParts[1].trim();
+
+        tasks.add(new DeadlineTask(description, deadline));
+        System.out.println("____________________________________________________________");
+        System.out.println(" Got it. Zappy boy added this task:");
+        System.out.println("   [D][ ] " + description + " (By: " + deadline + ")");
+        if (tasks.size() == 1) {
+            System.out.println(" Now you have 1 task in the list.");
+        } else {
+            System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void addEvent(String userCommand) {
+
+        String[] descParts = userCommand.split("event");
+        String[] eventParts = descParts[1].split("/from", 2);
+
+        if (eventParts.length != 2) {
+            System.out.println("Invalid event format. Please use '/from' to specify start and end time.");
+            return;
+        }
+
+        String description = eventParts[0].trim();
+        String[] timeParts = eventParts[1].split("/to", 2);
```

Comment:
U may want to use a better name for "timeParts" to be more descriptive and specific of the variable.